src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1287,4 +1287,13 @@
   <data name="PreferTypedStringBuilderAppendOverloadsRemoveToString" xml:space="preserve">
     <value>Remove the ToString call</value>
   </data>
+  <data name="PreferConstCharOverConstUnitStringInStringBuilderDescription" xml:space="preserve">
+    <value>StringBuilder.Append(char) is more efficient than StringBuilder.Append(string) when the string is a single character. When calling Append with a constant, prefer using a constant char rather than a constant string containing one character.</value>
+  </data>
+  <data name="PreferConstCharOverConstUnitStringInStringBuilderMessage" xml:space="preserve">
+    <value>Use StringBuilder.Append(char) instead of StringBuilder.Append(string) when the input is a constant unit string.</value>
+  </data>
+  <data name="PreferConstCharOverConstUnitStringInStringBuilderTitle" xml:space="preserve">
+    <value>Consider using StringBuilder.Append(char) when applicable.</value>
```

Comment:
Suggestion: `StringBuilder.Append(char)` -> `'StringBuilder.Append(char)'`

Reply:
In my PR, I got the suggestion to consider adding separate strings for VB and for C#. Should we consider this suggestion for all these messages as well?